scene/resources/visual_shader_nodes.cpp

```diff
@@ -7482,6 +7482,9 @@ String VisualShaderNodeMultiplyAdd::get_output_port_name(int p_port) const {
 }
 
 String VisualShaderNodeMultiplyAdd::generate_code(Shader::Mode p_mode, VisualShader::Type p_type, int p_id, const String *p_input_vars, const String *p_output_vars, bool p_for_preview) const {
+	if (OS::get_singleton()->get_current_rendering_method() == "gl_compatibility") {
+		return "	" + p_output_vars[0] + " = " + p_input_vars[0] + " * " + p_input_vars[1] + " + " + p_input_vars[2] + ";\n";
```

Comment:
Is there a risk that the input vars could be expressions? If so it might need parentheses.

Reply:
I'm not sure. I don't think so. I can't see special handling in case of expressions elsewhere, but I have updated with parentheses anyway